tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -68,6 +69,26 @@ describe(__filename, () => {
     ).toContain('You need to download Firefox to install this add-on.');
   });
 
+  it('renders a generic notice for non-Firefox browsers', () => {
+    _dispatchClientMetadata({
+      userAgent: userAgentsByPlatform.mac.chrome41,
+    });
+    const root = render({ reason: INCOMPATIBLE_NOT_FIREFOX });
+
+    expect(root.find('.AddonCompatibilityError').find(Notice)).toHaveProp(
+      'type', 'generic');
+  });
+
+  it('renders an error notice for other reasons browsers', () => {
```

Comment:
Should this name just be `renders an error notice for other reasons` (i.e., lose the `browsers` at the end)?

Reply:
Oops, thanks.